dds_web/static/swagger.yaml

```diff
@@ -631,6 +980,85 @@ components:
         checksum:
           type: string
           example: cheksum
+    User:
+      type: object
+      required:
+        - role
+        - unit
+        - email
+      properties:
+        role:
+          type: string
+          example: Researcher
+        unit:
+          type: string
+          example: UnitA
+        email:
+          type: string
+          example: researcher1@mailtrap.io
+        send_email:
+          type: boolean
+          example: true
+    UserInfo:
+      type: object
+      properties:
+        info:
+          type: object
+          properties:
+            email_primary:
+              type: string
+              example: unitadmin1@mailtrap.io
+            emails_all:
+              type: array
+              items:
+                type: string
+        is_admin:
+          type: boolean
+          example: true
+        name:
+          type: string
+          example: Unit Admin 1
+        role:
+          type: string
+          example: Unit Admin
+        username:
+          type: string
+          example: unitadmin_1
+    UserList:
+      type: object
+      properties:
+        active:
+          type: boolean
+          example: true
+        name:
+          type: string
+          example: Unit Admin 1
+        role:
+          type: string
+          example: Unit Admin
+        username:
+          type: string
+          example: unitadmin_1
+        email:
+          type: string
+          example: unitadmin1@mailtrap.io
+    InvitesList:
+      type: object
+      properties:
+        Created:
+          type: string
+          format: date-time
+        Email:
+          type: string
+          example: researcher1@mailtrap.io
+        Role:
+          type: string
+          example: Researcher
+        Projects:
+          type: array
+          items:
+            type: string
+          example: [project1]
```

Comment:
```suggestion
        Unit:
          type: string
          example: ""
```

Reply:
Ok, so it seems that the returned json is different if using superadmin or unitadmin token.

When unitadmin no unit field is returned. 
Superadmin returns the empty unit filed (which is a bit weird)
I will see if I can mark it as optional returned or something like that

Reply:
Yes, and it is a bit confusing, and as far as I can see it always returns an empty unit field for Superadmin, regardless 
of whether the invitation is connected to a unit or not. If it is connected, then only the project appears in the Projects field, but not the unit. This doesn't make a lot of sense, we either remove the unit field, or add a. value to it when relevant

Reply:
I created a task for it